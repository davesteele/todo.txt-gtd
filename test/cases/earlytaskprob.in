# AProj

This task should be in BProj @computer +BProj

# BProj
# The real BProj header

Existing BProj task @computer +BProj

# _None
