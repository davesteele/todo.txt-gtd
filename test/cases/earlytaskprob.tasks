
@computer
This task should be in BProj @computer +BProj
Existing BProj task @computer +BProj
