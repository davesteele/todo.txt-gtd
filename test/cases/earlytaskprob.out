# AProj


# BProj
# The real BProj header

Existing BProj task @computer +BProj

This task should be in BProj @computer +BProj

# _None
